glyphs-reader/src/font.rs

```diff
@@ -2993,20 +2993,26 @@ impl Font {
         if font.custom_parameters.propagate_anchors.unwrap_or(true) {
             font.propagate_all_anchors();
         }
+
+        // ensure that glyphs with components that have bracket layers
+        // also have bracket layers.
+        font.align_bracket_layers();
```

Comment:
(this call was missing before from load_from_string, so not just a simple git revert)